pkg/drivers/kvm/kvm.go

```diff
@@ -224,7 +214,7 @@ func (d *Driver) GetSSHHostname() (string, error) {
 
 // DriverName returns the name of the driver
 func (d *Driver) DriverName() string {
-	return constants.DriverKvm2
+	return "kvm2"
```

Comment:
can we make it a constant inside the driver?

Reply:
yep, done.

Reply:
But why

Reply:
@afbjorklund As those are the driver names I consider they good for a constant. To always have only one place to change it, even though now we don't have many. But as always, no strong feelings. and probably a too simple thing to start a big discussion, so all good to revert it to string only :)

Reply:
Well, at least it's not using the constants module. But the string is only used in 1 or 2 places ?

It's the not the most important thing, just thought it was an unnecessary step of indirection.